src/components/Logo/Logo.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Image from 'next/image';
+import logo from '../../../public/img/logo.png';
+
+type logoType = {
+    logoSize: number;
```

Comment:
`size` would be a better name for this prop since it already refers to the component that is already named `logo`. Naming it `logoSize` adds unnecessary redundancy.